tests/testthat/test_apa_print_anova.R

```diff
@@ -243,21 +243,21 @@ test_that(
     rm_afex_aov_output <- apa_print(rm_afex_aov$aov)
     expect_identical(rm_afex_aov_output, rm_aov_output)
 
-    rm_afex_aov_output <- apa_print(rm_afex_aov$Anova, correction = "none")
+    rm_afex_aov_output <- apa_print(rm_afex_aov$Anova, correction = "none", intercept = FALSE)
     expect_identical(rm_afex_aov_output, rm_aov_output)
 
-    rm_afex_aov_output <- apa_print(rm_afex_aov, correction = "none")
+    rm_afex_aov_output <- apa_print(rm_afex_aov, correction = "none", intercept = FALSE)
     expect_identical(rm_afex_aov_output, rm_aov_output)
 
 
     # DF corrections
-    rm_afex_anova.mlm_output <- apa_print(rm_afex_aov$Anova, correction = "GG")
+    rm_afex_anova.mlm_output <- apa_print(rm_afex_aov$Anova, correction = "GG", intercept = TRUE)
```

Comment:
Do we need to set `intercept = TRUE` in these tests? Shouldn't make any difference, no?

Reply:
Totally true, removed these changes.